packages/inference/src/providers/fal-ai.ts

```diff
@@ -28,4 +28,7 @@ export const FAL_AI_SUPPORTED_MODEL_IDS: ProviderMapping<FalAiId> = {
 		"THUDM/CogVideoX-5b": "fal-ai/cogvideox-5b",
 		"Lightricks/LTX-Video": "fal-ai/ltx-video",
 	},
+	"text-to-audio": {
```

Comment:
AFAIK, the inference.js client has a `text-to-speech` task, not `text-to-audio`: https://github.com/huggingface/huggingface.js/blob/2c759b54070b0fad02220f8159eaf52bda531b0b/packages/inference/src/tasks/audio/textToSpeech.ts

Reply:
true but i think the APIs are the same, so it should be quite easy to add! (single digit line of code change)

Reply:
yes I would also expect to be the same

```suggestion
	"text-to-speech": {
```